doc/man3/SSL_get_handshake_rtt.pod

```diff
@@ -0,0 +1,59 @@
+=pod
+
+=head1 NAME
+
+SSL_get_handshake_rtt
+- get round trip time for SSL Handshake
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ uint64_t SSL_get_handshake_rtt(const SSL *s);
+
+=head1 DESCRIPTION
+
+SSL_get_handshake_rtt() returns the round trip time for B<ssl>,
+for both he client and a server. If B<ssl> is NULL, 0
+is returned.
+
+This metric is returned in microseconds (us) as a uint64_t data type.
+
+=head1 NOTES
+
+This metric is created by taking two timestamps during the handshake and providing the
+absolute value difference betwwen the two times.
+
+When acting as the server, one timestamp is taken when the server is finished
+writing to the client. This id during the ServerHelloDone in TLS 1.3 and 
+ServerFinished in TLS 1.2. The other timestamp is taken when the server is done reading 
+the client's response. This is after the client has responded with ClientFinished.
+
+When acting as the client, one timestamp is taken when the client is finished writing 
+the hello and early data. The other is taken when client is done reading the server's response.
+This is after ServerFinished in TLS 1.3 and after ServerHelloDone in TLS 1.2.
+
+In addition to network propagation delay and network stack overhead, this metric also includes 
+some openssl processing on the end of both the client and server in all cases, as it is timing 
+how quickly the other endpoint can respond. The processing time is most often negligible, but 
+it is worth noting.
+
+It can be used for various performance metrics as well as possibly detecting the use of
+a web proxy.
+
+=head1 RETURN VALUES
+
+Returns a uint64_t value representing rtt in microseconds. 
+Returns 0 if the TLS handshake round trip time is not yet determined or if an error occurred in 
+calculation.
+
+=head1 COPYRIGHT
```

Comment:
Please add HISTORY section above this containing `SSL_get_handshake_rtt() was added in OpenSSL 3.2.0.`